README.md

```diff
@@ -10,10 +10,11 @@
 **Gra**ph **S**tatistics in **Py**thon is a package for graph statistical algorithms.
 
 - [Overview](#overview)
-- [Documentation](#documentation)
```

Comment:
This change breaks the table of contents FYI. IDK if you can do links and TOC at the same time

Reply:
What do you mean by "breaks"? I was suggesting that this link to the docs instead of moving you down the page to something that gives you a link to the docs. But would you rather have everything here just be TOC?

Reply:
I think I prefer it to be TOC only cause it doesn't make sense for some parts to behave like TOC and some not.

Tho we should put the `Tutorials` within `Documentation`

Thoughts?

Reply:
agree